lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -80,7 +90,7 @@ def install!
           # @return [Pathname] The absolute path of the Info.plist to use for an app host.
           #
           def app_host_info_plist_path
-            project.path.dirname.+("#{name}/#{name}-Info.plist")
+            project.path.dirname.+("#{group}").+("#{name}/#{name}-Info.plist")
```

Comment:
why is the group in the Xcode project being used as a part of the path?

Reply:
Should be `group.path` updated.